drivers/CoreLED/source/CoreLED.cpp

```diff
@@ -0,0 +1,47 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "CoreLED.h"
+
+using namespace leka;
+
+auto CoreLED::setColor(RGB color) -> void
+{
+	_color = color;
+}
+
+auto CoreLED::isOn() const -> bool
+{
+	return _is_on;
+}
+
+auto CoreLED::showColor() -> void
+{
+	uint8_t byte_zero  = 0x00;
+	uint8_t brightness = 0x32;
```

Comment:
```suggestion
	constexpr uint8_t byte_zero  = 0x00;
	constexpr uint8_t brightness = 0x32;
```

pourquoi 32 pour la brightness?

pourquoi `byte_zero`? `0x00` c'est pas assez clair?

Reply:
J'ai eu un souci justement, les passer en mode "nombre magique" fonctionne pas car il les considère comme des int, et non comme des unsigned int

Reply:
tu peux faire `std::to_array<uint8_t>({0x00, 0x00, 0x00, 0x00});`